index.html

```diff
@@ -716,20 +616,23 @@ <h3>
           <li>Remove |lock| from
           |document|.{{Document/[[ActiveLocks]]}}[|type|].
           </li>
+          <li>[=Queue a task=] on the [=screen wake lock task source=] to:
+            <ol>
+              <li>Set |lock|'s {{WakeLockSentinel/[[Released]]}} to `true`.
+              </li>
+              <li>[=Fire an event=] named "`release`" at |lock|.
```

Comment:
An event is already fired at the end of these steps. I think moving this into a task on the screen wake lock task source is the right move but it's probably best to just wrap all the steps in that since they're manipulating properties of the document.

Reply:
makes sense, yeah.